go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/fluid-cloudnative/fluid
 
-go 1.22.10
+go 1.23.0
```

Comment:
How about using Go version 1.22.10 to maintain consistency with the Dockerfile we are using?

Reply:
Done. Update to 1.23.7